### PR TITLE
Enable GPDB to dump with Netbackup client 7.6

### PR DESCRIPTION
**Pull request by Chibin**
Updating makefile to be able to run netbackup 7.6

Edit: We will do this in a future commit
Reverting -
Removing unnecessary libraries, that should be already in the system that can run netbackup.

### CODE REVIEW

File: gpAux/Makefile [hunk 1]
```diff
@@ -1137,19 +1137,6 @@ endif
...
-ifeq ($(BLD_ARCH),$(filter $(BLD_ARCH),rhel6_x86_64 rhel5_x86_64))
-	rm -f $(INSTLOC)/lib/libxbsa64.so;
-	rm -f $(INSTLOC)/lib/libnbclientcST.so;
-	rm -f $(INSTLOC)/lib/libnbbasecST.so;
-	rm -f $(INSTLOC)/lib/libvxcPBXST.so;
-	cp -fpr $(BLD_THIRDPARTY_LIB_DIR)/../Netbackup/nbu71 $(INSTLOC)/lib/;
-	cp -fpr $(BLD_THIRDPARTY_LIB_DIR)/../Netbackup/nbu75 $(INSTLOC)/lib/;
-endif
-
 copylibs : thirdparty-dist copy-nbu-libs
```
Comment (chrishajas) [inline gpAux/Makefile]: remove copy-nbu-libs reference

File: src/bin/pg_dump/cdb/Makefile [hunk 2]
```diff
@@ -100,6 +105,7 @@ install: all installdirs
 	$(INSTALL_PROGRAM) cdb_dump_agent$(X) $(DESTDIR)$(bindir)/gp_dump_agent$(X)
 	$(INSTALL_PROGRAM) cdb_restore_agent$(X) $(DESTDIR)$(bindir)/gp_restore_agent$(X)
 	$(INSTALL_PROGRAM) gpddboost$(X) $(DESTDIR)$(bindir)/gpddboost$(X)
+	$(INSTALL_PROGRAM) libgpbsa76.so$(X) $(DESTDIR)$(libdir)/nbu76/lib/libgpbsa.so$(X)
```
Comment (chrishajas) [inline src/bin/pg_dump/cdb/Makefile]: Does this need to be removed in  `clean distclean maintainer-clean`

File: gpAux/Makefile (hunk 1 above)
Comment (Chibin) [inline gpAux/Makefile]: Addressed, thanks!

**Comment by Chibin** [comment]
Reverting -
Removing unnecessary libraries, that should be already in the system that can run netbackup.

We will do this in a future commit

**Pull request merged**